src/main/java/school/lemon/changerequest/java/introduction/hw2/Task8.java

```diff
@@ -3,6 +3,28 @@
 public class Task8 {
 
     public static int[] evenOdd(int[] arr) {
-        return null;
+        if(arr == null)
+            return arr;
+
+        int []result = new int[arr.length];
+        int i = 0;
+        for(int a:arr)
```

Comment:
Please format your code with default IDEA code formatter before submitting it. Again, it's related to Java Code Conventions and code style. You should not use `C/C++`-like code style, while you're using Java.